### ISSUE TITLE
1.18.2 structures spawning with config set to false

### ISSUE THREAD

**Issue by zachsai** [Tictim/Paraglider#48]
![image](https://user-images.githubusercontent.com/63996164/162555692-7b69a1ea-557a-493b-a5c5-84a94dd6bb86.png)
![image](https://user-images.githubusercontent.com/63996164/162555953-93300b90-92eb-4236-961f-9da6fdd4feab.png)
![image](https://user-images.githubusercontent.com/63996164/162555704-1c24d6fb-b87a-477f-ab74-300be57b7306.png)

With structures set to false, they are still spawning

**Comment by Tictim** [comment]
Yeah, the option wouldn't be able to block custom structure spawning anymore due to the structures being registered with datapacks. For now, as a workaround, you could disable them with datapacks, by emptying `has_structure` tags attached to each structures.

Also the structure at second screenshot shouldn't spawn in nether. It has been checked during the test so I'm not sure what's happening there. Do you have any worldgen mods installed?

**Comment by evanbones** [comment]
The paraglider statues spawn in Terralith, Incendium, and Nullscape biomes because they don't have biome tags as they're datapacks, not mods. This can be fixed with biometweaker

**Comment by Tictim** [comment]
Closing as stale.
